test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
Pretty sure that version numbers are numeric :), even if they have designed names, at least according to the [semver](http://semver.org/#spec-item-11) or any other number schemas really, since i'm not sure how you can describe a range without numeric values.

Most of the text style guides like [APA](https://en.wikipedia.org/wiki/APA_style) & [AMA](https://en.wikipedia.org/wiki/AMA_Manual_of_Style), which is hardly an "English prose", use variation of dash without the spaces, even HYPHEN-MINUS (specifically "HYPHEN" btw, not "HYPEN") and EN DASH are:

> equally "correct", and each is the preferred style in some style guides 

but always, always without a spaces.

Rules you devised in that comment are great, they are just deviate from most of guides for natural languages (specifically including technical documents) around the world (not only in English actually). I don't think we need to invent a new language or text style guide for this, just to use already created one.

Like if you invent formal specification to illustrate your point, it doesn't make it widely acceptable idea. Wicked good way to present your point though :).

Instead, I would prefer to use something that everybody use, if you don't like form of dashes used in academic papers (or in typography as a whole), we can replace it with the word "to" with spaces like in [SI](http://physics.nist.gov/cuu/pdf/sp811.pdf) guide of National Institute of Standards and Technology, which bears insignia of U.S. Department of Commerce